client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

```diff
@@ -3,51 +3,51 @@ import { LaunchpadFlowTaskList, Task } from './types';
 export const tasks: Task[] = [
 	{
 		id: 'setup_newsletter',
-		isCompleted: true,
+		completed: true,
```

Comment:
using the built-in `completed` and `disabled` props we can achieve every possible combination of a task's status